orc8r/cloud/go/obsidian/swagger/v1/client/README.md

```diff
@@ -0,0 +1,17 @@
+# Orchestrator REST API v1
+
+This directory contains v1 of the Orchestrator REST API.
+
+We do not provide any guarantees about the stability of the REST API. This will likely change in the future.
+
+## `swagger.yml` file
+
+This file contains the full specification of the Orchestrator REST API.
+
+The spec file is automatically generated as an aggregation of the `swagger.v1.yml` files throughout the codebase.
+
+## `client` directory
+
+This directory contains Golang bindings for the Orchestrator REST API.
```

Comment:
Is there a section on how we generate these bindings ? Do we generate it in CI ?

Reply:
Updated to address -- `build.py --generate` aka the `fullgen` make target